src/components/chip/chip.tsx

```diff
@@ -75,12 +88,30 @@ export class Chip
   /** Specifies the size of the component. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
+  /** Accessible name for the component. */
+  @Prop() label: any;
```

Comment:
@macandcheese Concur on a required `label`, unless there is a way we could assign the text as a `label`?

Reply:
Text is just slotted, so it can't be relied on to be present - icon only chip for example.

Reply:
And yes, thanks for the reminder on the icon only `chip`, concur with the required `label` prop. 👍🏻

Reply:
As discussed with @jcfranco - `label` will not be required, and `value` will remain required to avoid breaking changes and the need for translations internally for components that render `chip` components of their own - Alert, Combobox, Rating. 

These being required or not is pretty inconsistent across all components, so any patterns we end up moving towards can be applied to Chip at that time.

Reply:
Is the bigger concern around breaking changes for the requirement? We'll definitely need to doc the importance of the `label` prop if its not required, in particular for situations where there's only an icon present.

Reply:
cc @jcfranco 🤔

Reply:
> Is the bigger concern around breaking changes for the requirement? We'll definitely need to doc the importance of the label prop if its not required, in particular for situations where there's only an icon present.

Correct. Making it required will cause TS errors to start showing up if it wasn't already being provided for existing chips. 

> These being required or not is pretty inconsistent across all components, so any patterns we end up moving towards can be applied to Chip at that time.

💯, we can start tracking these and batch update them for whenever we have a future breaking release in the works.